crud.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.DBAuthor(name=author.name, bio=author.bio)
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+
+def get_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.DBAuthor).offset(skip).limit(limit).all()
```

Comment:
Ensure that `models.DBAuthor` is defined and that the query correctly retrieves author records.